x/dex/keeper/msgserver/msg_server_place_orders_test.go

```diff
@@ -26,6 +26,8 @@ func TestPlaceOrder(t *testing.T) {
 				OrderType:         types.OrderType_LIMIT,
 				PriceDenom:        keepertest.TestPriceDenom,
 				AssetDenom:        keepertest.TestAssetDenom,
+				ContractAddr:      TestContract,
```

Comment:
@codchen looks like contract adder is in 2 places here:
1. MsgPlaceOrders: https://github.com/sei-protocol/sei-chain/blob/master/x/dex/keeper/msgserver/msg_server_place_orders_test.go#L17
2. Orders (this change)
Is it possible to remove this from Orders and infer it from MsgPlaceOrders (or remove from MsgPlaceOrders)? I assume they should be the same. Would like to remove duplicate data if possible.

Reply:
Account and contract address are already inferred here: https://github.com/sei-protocol/sei-chain/pull/175/files#diff-555639b7dfe25d62ab5dc8f8746b1588a98740aef6e622539386fcaec4508c04R89 and so we don’t need to validate those two fields in the validation method